tools/saqc/saqc.xml

```diff
@@ -0,0 +1,48 @@
+<tool name="SaQC" id="saqc" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.01">
+  <description>quality control pipelines for environmental sensor data</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"/>
```

Comment:
Kannst du das hizufuegen https://github.com/Helmholtz-UFZ/galaxy-tools/blob/61a3d9a20a9a90d551dd5f7503be781dc28f4b75/tools/tox_tools/baseline_calculator/qsar1.xml#L4 .. ohne das department und mit direktem link zum ufz.

Reply:
Oder mache `Helmholtz Centre for Environmental Research - UFZ, Research Data Management` mit https://www.ufz.de/index.php?en=45348